lookbook/docs/components/page-header.md.erb

```diff
@@ -28,6 +28,29 @@ There are minor adjustments on how the composition of this component is defined
 - A special behaviour occurs when the page can be saved (or saved as) as one of the PageHeader actions will be used to display an informational text and a link to perform the action.
 - Work packages details are the only exceptional case, where the breadcrumb will be used to display the parent work package instead of the navigational hierarchy.
 
+## **Breadcrumb usage**
+By default, every PageHeader should include breadcrumbs. They are a core part of the global navigation in OpenProject and must be present on almost all pages.
+
+There are a few exceptional cases where the PageHeader may be rendered without breadcrumbs (e.g., my page and the project overview page).
+Only use this variant if there is a strong UX reason.
```

Comment:
I guess we should write down, why the my page and the project overview page have strong UX reasons not do show the breadcrumb. Further, "strong UX reasons" is very vague and leaves the door wide open for everybody to argue that he/she had strong reasons to do it.